go.mod

```diff
@@ -68,6 +67,8 @@ require (
 	golang.org/x/sys v0.0.0-20210315160823-c6e025ad8005
 	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
-	k8s.io/api v0.0.0-20190620084959-7cf5895f2711
+	k8s.io/api v0.20.1
 	k8s.io/apimachinery v0.20.4
 )
+
+replace github.com/syndtr/gocapability => github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2
```

Comment:
I assume this fixes the `in container` testing, right?

I'm a bit afraid of this trick since it's not sustainable on the long run.  Downstream packages like Debian will have troubles, and we don't get updates.

Reply:
https://github.com/opencontainers/runc/commit/6dfbe9b80707b1ca188255e8def15263348e0f9a is part of runc rc93. What is the reason to still pin this dependency?

Reply:
It's causing issue in the in-container tests. @giuseppe is about to create a separate PR with the new c/common release to address these.

It's a temporary fix to unblock bumping c/storage et al.

Reply:
fixed with https://github.com/containers/podman/pull/9762